docs/src/api/class-frame.md

```diff
@@ -97,6 +97,33 @@ with sync_playwright() as playwright:
     run(playwright)
 ```
 
+```csharp
+using Microsoft.Playwright;
+using System;
+using System.Threading.Tasks;
+
+class FrameExamples
+{
+    public static async Task Main()
+    {
+        using var playwright = await Playwright.CreateAsync();
+        var browser = await playwright.Firefox.LaunchAsync();
+        var page = await browser.NewPageAsync();
+
+        await page.GoToAsync("https://www.bing.com");
+        DumpFrameTree(page.MainFrame, string.Empty);
+        await browser.CloseAsync();
```

Comment:
Why not doing `await using`? we would be teaching people new stuff :)

Reply:
This applies to the rest of the open PRs

Reply:
Yup, I've changed all three occurrences.